transport-native-io_uring/src/main/java/io/netty/channel/uring/IOUringDatagramChannel.java

```diff
@@ -326,75 +323,80 @@ protected void doClose() throws Exception {
     }
 
     final class IOUringDatagramChannelUnsafe extends AbstractUringUnsafe {
-        private ByteBuf readBuffer;
-        private boolean recvMsg;
-
         // These buffers are used for msghdr, iov, sockaddr_in / sockaddr_in6 when doing recvmsg / sendmsg
         //
         // TODO: Alternative we could also allocate these everytime from the ByteBufAllocator or we could use
         //       some sort of other pool. Let's keep it simple for now.
```

Comment:
Apart from possibly being more complicated (I don't think it would be too bad), do you envisage any other downside from using existing pooled bytebufs for this (and iovecs generally)?

Reply:
I will need to do some experiments. But also the "housekeeping" will be a bit more complicated + sometimes people not use pooling